integrations/clickup/.port/resources/port-app-config.yaml

```diff
@@ -0,0 +1,49 @@
+createMissingRelatedEntities: true
+deleteDependentEntities: true
+resources:
+  - kind: team
+    selector:
+      query: 'true'
+    port:
+      entity:
+        mappings:
+          blueprint: '"clickupTeam"'
+          identifier: .id | tostring
+          title: .name
+          properties:
+            url: "\"https://app.clickup.com/\" + .id + \"/home\""
+            members: '[.members[].user.email]'
+  - kind: project
+    selector:
+      query: 'true'
+    port:
+      entity:
+        mappings:
+          blueprint: '"clickupProject"'
+          identifier: .id | tostring
+          title: .name
+          properties:
+            url: "\"https://app.clickup.com/\" + .team_id + \"/v/l/li/\" + .id"
+            startDate: .start_date | tonumber / 1000 | todate
+            endDate: .due_date | tonumber / 1000 | todate
+            totalIssues: .task_count
+          relations:
+            team: .__team_id
```

Comment:
too specific, should have added an option to see all the team object, this will allow extensibility in the future